### DISCUSSION TITLE
Bug: `Tools → Proxy Settings → Override Mac OS Proxy` setting is not preserved after app restart

### DISCUSSION THREAD

**Discussion by igorwojda** [ProxymanApp/Proxyman#2151] [General]
Reproduce:
1. Turn off `Tools → Proxy Settings → Override Mac OS Proxy`
2. Restart The App
3. Option is turned on

Context:
As mobile dev I am only using Proxyman to review traffic from external devices, so I would like to have this option preserved

**Comment by NghiaTranUIT** [comment]
You can turn this setting off in the Settings -> Un-check "Auto override system proxy at launch"

<img width="552" alt="Screenshot 2024-10-09 at 8 16 32 AM" src="https://github.com/user-attachments/assets/e799868a-c95e-4781-95fb-27f015e0a903">
